examples/testapp/src/components/RpcMethods/shortcut/signMessageShortcuts.ts

```diff
@@ -130,6 +130,41 @@ const ethSignTypedDataV4Shortcuts: (chainId: number) => ShortcutType[] = (chainI
       address: ADDR_TO_FILL,
     },
   },
+  {
+    key: "Grant Permission",
+    data: {
+      message: {
+        domain: {
+          name: "Spend Permission Manager",
+          version: "1",
+          chainId: chainId,
+          verifyingContract: "0x4ba47ee994d7fd63e8e8029431c35afa8487b8a5"
+        },
+        types: {
+          SpendPermission: [
+            {name: "account", type: "address"},
+            {name: "spender", type: "address"},
+            {name: "token", type: "address"},
+            {name: "start", type: "uint48"},
+            {name: "end", type: "uint48"},
+            {name: "period", type: "uint48"},
+            {name: "allowance", type: "uint160"}
+          ]
+        },
+        primaryType: "SpendPermission",
+        message: {
+          account: '0xd4e17478581878A967aA22d45a5158A9fE96AA08',
```

Comment:
is this the account that is granting the permission? should it be unique to the connected address?

Reply:
Yeah, this would need to be the connected account, and the spender would need to be the signer the dapp sets up. This means the message is not viable on-chain, but I think it could be fine as an example message? Any ideas to imrpove here?

Reply:
Good to know, that makes sense. I think this is fine for now. As we start working on linked accounts we could expand this to set the spender as the linked account and the address as the users global account. 

Can you run yarn link from the example/testapp directory. Looks like its not working currently and ends up breaking deploys. I'll start working on a fix for that.